packages/convert-svg-core/src/Converter.js

```diff
@@ -278,6 +278,19 @@ html { background-color: ${provider.getBackgroundColor(options)}; }
 
     await writeFile(tempFile.path, html);
 
+    this[_page].on('requestfinished', (msg) => {
+      const resource = msg.url();
+      const statusCode = msg.response().status();
+      if (statusCode >= 400) {
+        throw new Error(`A request to ${resource} failed with ${statusCode}`);
+      }
+    });
+
+    this[_page].on('requestfailed', (msg) => {
+      const resource = msg.url();
+      throw new Error(`Unable to load resource: ${resource}`);
+    });
```

Comment:
These errors are being thrown within event handlers, so may not be caught by the calling code in the expect async manner. Not sure how easy/clean this is going to be to implement though.

We also need tests added to cover these scenarios.